roomranger/frontend/src/App.js

```diff
@@ -1,81 +1,95 @@
+        <div className={`slider ${isDarkMode ? 'dark' : 'light'}`} />
+         <span className="mode-text">{isDarkMode ? 'Light' : 'Dark'}</span>
+      </div>
       <Router>
-        <Routes>
-          {/* <Route element={<HomeNav />}> */}
-          {/* <Route element={<RouteHome />}>
-              <Route path='/' element={<Home />} />
-              <Route path='login' element={<LoginPage />} />
-              <Route path='register' element={<Register />} /> 
-           </Route> */}
-          {/* </Route> */}
-          <Route path='/' element={<RouteHome />}>
-            {/* <Route index element={<Register />} /> */}
-            <Route path='login' element={<LoginPage />} />
-          </Route>
-          <Route path="/landing" element={<NavBar />} > 
-            {/* <Route path='assignedrooms' element={<RouteHome />}> */}
-              <Route index element={<HomeManager />} />
-              <Route path="assignroomform" element={<AssignRoom />} />
-              <Route exact path="editassignedroom/:id" element={<EditAssignedRoom />} />
-            {/* </Route> */}
-            <Route path="rooms" element={<RouteHome />}>
-              <Route index element={<HomeRoom />} />
-              <Route path="addroom" element={<AddRoom />} />
-              <Route path="editroom/:id" element={<EditRoom />} />
-              <Route path="viewroom/:id" element={<ViewRoom />} />
-            </Route>
-            <Route path="attendants" element={<RouteHome />}>
-              <Route index element={<AttendantListComponent />} />
-              <Route path="add" element={<AddAttendant />} />
-              <Route path="update/:id" element={<Update />} />
-              <Route path="profile/:id" element={<Profile />} />
-              <Route exact path="*" element={<ErrorComponent />} />
-            </Route>
-            <Route path="leave" element={<RouteHome />} >
-              <Route index element={<LeaveList />} />
-              <Route path="edit/:id" element={<EditRequest />} />
-              <Route path="form" element={<LeaveForm />} />
-            </Route>
-          </Route>
-          {/* <Route exact path="/attendant" element={<HomeAttendant />} />  */}
+         <Routes>
+                  {/* <Route element={<HomeNav />}> */}
+                  {/* <Route element={<RouteHome />}>
+                      <Route path='/' element={<Home />} />
+                      <Route path='login' element={<LoginPage />} />
+                      <Route path='register' element={<Register />} />
+                   </Route> */}
+                  {/* </Route> */}
+                  {/* <Route path='/' element={<RouteHome />}> */}
+                    <Route path ='/'element={<Home />} />
+                    <Route path='Register' element={<Registration />} />
+                    <Route path='login' element={<LoginPage />} />
+                    <Route path='about' element={<About />} />
```

Comment:
How is About show now? Please send us a screenshot! Will it show in everypage?

Reply:
sure